### PR TITLE
Added arduino workshop event

### PR DESCRIPTION
**Pull request by evykassirer**

### CODE REVIEW

File: content/F2015/F15-event-arduino-workshop.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+until we sell out.
+
+## Workshop Details ##
+
+*This is a [women-only]({filename}/pages/faq.md) event.*
+
++ **Who:** Melissa Tedesco, mentors, and registered attendees
++ **What:** Arduino Day Hardware Workshop
++ **Where:** Details will be emailed to participants
++ **When:** Saturday, Nov 7
```
Comment (evykassirer) [inline content/F2015/F15-event-arduino-workshop.md]: has a time been decided?

**Comment by evykassirer** [comment]
Thanks to @ehashman for a great blurb from last time that I was able to reuse a lot of :)

File: content/F2015/F15-event-arduino-workshop.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+with an Arduino!  The cost of the workshop is just $10, which contributes 
+towards the price of an Arduino, and you'll get to take whatever you've built 
+home with you. We'll be teaching you how to program LEDs and other sensors with 
+the board using an IDE so that you can get started on your very own project!"
+
+Registration is very limited and first-come first-served, so sign up as soon as
+possible! We'll be accepting registrations until the workshop or until we sell 
+out. [Register here](https://goo.gl/KPN4nf)
+
+You can register for the workshop and pay the $25 fee in the MathSoc
```
Comment (evykassirer) [inline content/F2015/F15-event-arduino-workshop.md]: ah, crap

**Comment by ehashman** [comment]
I'm pretty sure the blurb was mostly from Lux, but no problem :) Please squish down that second commit.

**Comment by evykassirer** [comment]
will do after I add the time tomorrow! and if no one else has suggested changes
thanks for the quick reply ^^

File: content/F2015/F15-event-arduino-workshop.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+title: Arduino Workshop
+Date: 2015-10-22
+Category: Events
+Tags: workshop, hardware
+Slug: arduino-workshop-F15
+Author: Evy Kassirer
+Summary: Come learn about digital hardware and Arduino programming in this women-only workshop for beginners!
+
+We're pleased to announce Melissa Tedesco's hardware workshop! Melissa this
```
Comment (evykassirer) [inline content/F2015/F15-event-arduino-workshop.md]: sigh need to get rid of 'this'
will do after my interviews

File: content/F2015/F15-event-arduino-workshop.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+team of mentors will introduce you to building prototypes with digital hardware 
+and the software needed to interact with it.
+
+Do you know how to program, but don't even know where to start with the
+hardware? Well, never fear, because WiCS is here! On November 7th, we will be
+running a workshop on how to put together hardware components and program them
+with an Arduino!  The cost of the workshop is just $10, which contributes 
+towards the price of an Arduino, and you'll get to take whatever you've built 
+home with you. We'll be teaching you how to program LEDs and other sensors with 
+the board using an IDE so that you can get started on your very own project!"
```
Comment (ehashman) [inline content/F2015/F15-event-arduino-workshop.md]: There's a stray " at the end of the line here.

File: content/F2015/F15-event-arduino-workshop.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+out. [Register here](https://goo.gl/KPN4nf)
+
+## Workshop Details ##
+
+*This is a [women-only]({filename}/pages/faq.md) event.*
+
++ **Who:** Melissa Tedesco, mentors, and registered attendees
++ **What:** Arduino Day Hardware Workshop
++ **Where:** Details will be emailed to participants
++ **When:** Saturday, Nov 7 10:30&ndash;4:30PM
```
Comment (ehashman) [inline content/F2015/F15-event-arduino-workshop.md:34]: This is pretty long so you may want to mention free lunch?

File: content/F2015/F15-event-arduino-workshop.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+hardware? Well, never fear, because WiCS is here! On November 7th, we will be
+running a workshop on how to put together hardware components and program them
+with an Arduino!  The cost of the workshop is just $10, which contributes 
+towards the price of an Arduino, and you'll get to take whatever you've built 
+home with you. We'll be teaching you how to program LEDs and other sensors with 
+the board using an IDE so that you can get started on your very own project!"
+
+Registration is very limited and first-come first-served, so sign up as soon as
+possible! We'll be accepting registrations until the workshop or until we sell 
+out. [Register here](https://goo.gl/KPN4nf)
```
Comment (ehashman) [inline content/F2015/F15-event-arduino-workshop.md]: After looking at this live, maybe also put a '.' after "here".

**Comment by ehashman** [comment]
LGTM.

**Pull request merged**